dashboard-ui/src/components/ActionsCompare/decisionMakerDash.jsx

```diff
@@ -0,0 +1,117 @@
+import React from "react"
+import { ListGroup, Card } from 'react-bootstrap'
+class DecisionMakerDash extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            htmlFileContent: null,
+            doc: null,
+            dashInfo: null
+        };
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props.decisionMaker !== prevProps.decisionMaker) {
+            this.setState({dashInfo: null})
+            this.fetchData(this.props.decisionMaker);
+        }
+    }
+
+    fetchData() {
+        if (this.props.decisionMaker && this.props.isHuman) {
+
+            const htmlFilePath = process.env.PUBLIC_URL + `/jungleExampleData/${this.props.decisionMaker === "human1" ? "jungle1" : "jungle2"}.html`;
```

Comment:
I know I mentioned this somewhere else, but ideally the data that is in these html files is in the JSON files and we then ingest that into the mongo database and then use an endpoint here to get that data.